snap/hooks/configure

```diff
@@ -34,13 +34,16 @@ sample_du_config=$(
     cat << END_OF_JSON
 {
   "schemaVersion": "1.1",
+  "aduShellFolder": "/snap/deviceupdate-agent/current/usr/lib/adu",
```

Comment:
non-snap agent uses /usr/bin for adu-shell so it's inconsistent this is under usr/lib/adu:
see https://github.com/Azure/iot-hub-device-update/blob/0f041bfc6cd25593ed61d58e4113b2a4dad7394f/packages/debian/postinst#L23